### ISSUE TITLE
Auto-merge Dependabot PRs

### ISSUE THREAD

**Issue by cnunciato** [pulumi/docs#5980]
Our current use of the `automerge` workflow doesn't work for Dependabot PRs (due to missing secrets), so we might want to run a scheduled job of some sort to recognize and merge them once they pass all checks required otherwise.

**Comment by susanev** [comment]
@sean1588 if this is a quick task, feel free to pick it up whenev

**Comment by sean1588** [comment]
ok cool. thanks @susanev!

**Comment by thoward** [comment]
Per @sean1588 this is still an issue, but extremely low priority as we can just merge these manually as needed. We have fixed similar issues that are run as the `pulumi-bot` user, but in this case, these are run under a different user. So, we'd either need to fix in the same way that we did for `pulumi-bot` or run these under `pulumi-bot` account. 

Since this has been on the backlog for roughly 2.5 years, and feels very low priority, should we close this as `wont-fix` or leave this here and prioritize fixing this paper cut within the next two quarters? Engineering effort seems small, and the impact is that it's one less special-case thing for docs automation operators to remember/keep track of as part of daily manual maintenance.
